### PR TITLE
Maps 3: Modifications to Existing files

### PR DESCRIPTION
**Pull request by PrinceUmbrella**
-> Created a new class to store the markers data. Each data type has latitude, longitude, and Content by the user.
-> Then Modified Datastore to handle and store this data type

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -83,10 +82,48 @@ public void storeMessage(Message message) {
...
+      } catch (Exception e) {
+        System.err.println("Error reading marker.");
+        System.err.println(entity.toString());
+        e.printStackTrace();
+      }
+    }
+    return markers;
+  }
+
+  public void storeMarker(UserMarker marker) {
```
Comment (minhkhuu10) [inline src/main/java/com/google/codeu/data/Datastore.java:121]: Can you please explain what does this part of the code does to me?

File: src/main/java/com/google/codeu/servlets/UserMarkerServlet.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        List<UserMarker> markers = datastore.getMarkers();
+        Gson gson = new Gson();
+        String json = gson.toJson(markers);
+
+        response.getOutputStream().println(json);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) {
```
Comment (minhkhuu10) [inline src/main/java/com/google/codeu/servlets/UserMarkerServlet.java:43]: Wow this project is very complicated. Kudos to you for finishing it.

File: src/main/java/com/google/codeu/data/UserMarker.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class UserMarker {
+
+    private double lat;
```
Comment (minhkhuu10) [inline src/main/java/com/google/codeu/data/UserMarker.java:5]: Can you please explain what lat, lng and content variables are please?

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (PrinceUmbrella) [reply src/main/java/com/google/codeu/data/Datastore.java:121]: This code stores the markers created on the map and loads it when it is open

File: src/main/java/com/google/codeu/data/UserMarker.java (hunk 3 above)
Comment (PrinceUmbrella) [reply src/main/java/com/google/codeu/data/UserMarker.java:5]: Lat -> Latitude
Lng -> Longitude
content -> users message on the marker

File: src/main/java/com/google/codeu/servlets/UserMarkerServlet.java (hunk 2 above)
Comment (PrinceUmbrella) [reply src/main/java/com/google/codeu/servlets/UserMarkerServlet.java:43]: Thank you Minh

**Review by minhkhuu10 — APPROVED** [review]

**Pull request merged**